documentation/latest/operations/troubleshooting.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: documentation
+---
+
+This document describes how to troubleshoot Flotta in different deployment
+modes. It focuses on a full deployment; If you are  looking for a simple way to
+experiment, we highly recommend trying out the Getting Started Guides instead.
+
+This guide assumes that you have some knowledge of all components and concepts.
+
+## Flotta operator & API Server
+
+### Edgedevices are not created when device is on.
+
+This maybe is because
+[AUTO_APPROVAL_PROCESS](deployment_options.md#auto_approval_process) is set to
+false. When this parameter is enabled on the operator, devices can only enrol,
+but a human needs to approve the device to land to a specific namespace, labels
+or deviceSet.
+
+### Devices are getting 401 responses
+
+This can be normal, mainly because some heartbeat process is executed before the
+system enrols into the cluster. To validate that your certificates are correct,
+you need to check the certs used, mainly using the following `openssl` command:
+
+```shell
+$ openssl x509 -noout -in /tmp/cert.pem --subject
```

Comment:
`/tmp/cert.pem` refers on to dev-mode. would you like to add the path on ostree-based device?